mateo-mustapic/assignments/project7/js/users.js

```diff
@@ -30,13 +29,16 @@ function search(){
     }
 }
 /* Sort the list of users */
-$(document).ready(function () {
+function sortUser(){
```

Comment:
Variables should not be in global scope, unless absolutely necessary. You change replaces a good pattern of sandboxing to ```.ready() ``` with more variables in global scope.